### PR TITLE
Pedantic tweaks

### PR DESCRIPTION
**Pull request by sindresorhus**

### CODE REVIEW

File: content.js [hunk 1]
```diff
@@ -62,8 +62,8 @@ const addStyles = () => {
 
 const showRecordingAlert = () => {
 	return SweetAlert.fire({
-		title: 'Recording...',
-		text: 'When you\'re done, click "Send" or press Enter.',
+		title: 'Recording…',
+		text: 'When you’re done, click “Send” or press Enter.',
```
Comment (stephenmathieson) [inline content.js:66]: I was about to open a PR just for this ❤️

**Comment by vadimdemedes** [comment]
Thanks!

File: content.js (hunk 1 above)
Comment (vadimdemedes) [reply content.js:66]: Heeey @stephenmathieson, long time no see!

Comment (stephenmathieson) [reply content.js:66]: 👋

**Pull request merged**